src/core/container.rs

```diff
@@ -84,13 +84,13 @@ where
     /// [`Image`]: trait.Image.html
     /// [`arguments`]: trait.Image.html#associatedtype.Args
     pub fn image(&self) -> &I {
-        &self.image
+        self.runnable.image()
     }
```

Comment:
By separating the `Args` from `Image`, I think we will need to expose `Image::Args` from a container as well.

That should also fix the ignored test with Bitcoin-Core's auth.

Reply:
WDYT of current approach?